### ISSUE TITLE
a11y minor issues on developer.microsoft.com

### ISSUE THREAD

**Issue by ramanova** [OfficeDev/office-ui-fabric-react#10261]
# Environment Information

- **Package version(s)**: (the current one on developer.microsoft.com demo)
- **Browser and OS versions**: (MacOS, Chrome Version 76.0.3809.100 (Official Build) (64-bit))

### Describe the issue:
A list of elements that have minor A11y issues according to Keros testing tool:

  https://developer.microsoft.com/en-us/fabric#/controls/web/combobox Disabled 
ComboBox (Elements must have sufficient color contrast) 

https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Dropdown (Elements must have sufficient color contrast) 

https://developer.microsoft.com/en-us/fabric#/controls/web/label
Elements must have sufficient color contrast

https://developer.microsoft.com/en-us/fabric#/controls/web/rating
id attribute value must be unique

https://developer.microsoft.com/en-us/fabric#/controls/web/slider
Elements must have sufficient color contrast

https://developer.microsoft.com/en-us/fabric#/controls/web/spinbutton
Required ARIA attributes must be provided

Split Button (Elements must only use allowed ARIA attributes) 

Custom Split Button (Buttons must have discernible text)

**Comment by maxwellred** [comment]
@ramanova I have confirmed that disabled colors is not and issue we are addressing and I believe the discernible text issue is a Keros issue. When I inspect the split button role attributes in code and in dev tools it doesn't show the issue that Keros calls out `aria-roledescription="split\ button"` so it might be worth opening an issue on their end. Lastly, the duplicate IDs I'm currently working on now and should have in PR shortly.

**Comment by smoralesd** [comment]
This accessibility insights web issue may be related:
https://github.com/microsoft/accessibility-insights-web/issues/981

**Comment by maxwellred** [comment]
microsoft/accessibility-insights-web#981

**Comment by maxwellred** [comment]
Hi @ramanova, It looks like everything is merged in. Thanks again for opening the issue and helping the project! If this covers everything we'd really appreciate having you close the issue, thanks!
